tests/consensus_tests/auth_tests/test_dashboard_ui_auth.py

```
@@ -0,0 +1,86 @@
+            "QDRANT__SERVICE__UI_PASSWORD": "s3cret-pass",
+        },
+    )
+
+    try:
+        yield peer_api_uris, peer_dirs, bootstrap_uri
+    finally:
+        kill_all_processes()
+
+
+def test_dashboard_requires_login(dashboard_ui_auth_cluster):
+    response = requests.get(f"{REST_URI}/dashboard", allow_redirects=False)
```

Ruff (0.15.15)

[error] Probable use of `requests` call without timeout

(S113)